docs/contributing/releasing.md

```diff
@@ -3,26 +3,46 @@
 > :memo: The versioning scheme we use is [SemVer](http://semver.org/). Note that until
 > we agree we're ready for v1.0.0, we will not increment the major version.
 
-1. Ensure all desired features are merged to `main` branch and `CHANGELOG.md` is updated.
+1. Ensure all desired features are merged to `main` branch and `CHANGELOG.md` is
```

Comment:
Should note that we needed to remove the brackets from `[Unreleased]` in order for `bumpy-my-version` to update the release for the CHANGELOG?

Reply:
Good catch, the brackets shouldn't have been there in the first place :)